### DISCUSSION TITLE
Print/export Decisions

### DISCUSSION THREAD

**Discussion by YoeriVD** [structurizr/lite#40] [Ideas]
Decisions are often passed around for review or reference, and it is not always straightforward to provide access to structurizr.

In these cases (which happen frequently) it would be very convenient to export a decision as a pdf.

I tried:

- Browser print as pdf (looks horrible)
- structurizr/puppeteer (does not work with decisions/pdf)
- generic markdown to pdf tools (not able to render embedded diagrams)

I am curious to hear thoughts on this, or perhaps suggestions for a workaround.

**Comment by simonbrowndotje** [comment]
I'd recommend publishing the workspace to an on-premises installation or the cloud service ... both are designed for distributing workspace content to a wider audience. You may also want to consider https://github.com/avisi-cloud/structurizr-site-generatr ... this will generate you an offline website, which includes decisions.

**Reply by YoeriVD** [reply]
I use both, but that still requires people to have access to structurizr in a browser. 

In this case I need to email a document, along with any ADR's for review.
Even if I convince someone to use the website, that still lacks the simple "forward" option an email offers.

In such cases, I really wish for a simple "print pdf" experience. 

some `@media print` css that hides the buttons would already suffice, as the browser will take care of the pdf.

**Reply by simonbrowndotje** [reply]
> some @media print css that hides the buttons would already suffice, as the browser will take care of the pdf.

Done.

**Reply by YoeriVD** [reply]
Not gonna lie, the deadline is tomorrow so that would be close to a lifesaver! :D
